app/controllers/api/v1/goods_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::GoodsController < ApplicationController
 
   def show
-    good = Good.find(params[:id])
+    good = Good.find_by_id(params[:id])
     if good
```

Comment:
@DavidKnott `find_by` returns nil instead of an error if the record doesn't exist. `find_by_id` is Rails magic for `find_by` with an attribute in the schema.